media-gfx/openvdb/files/openvdb-7.1.0-0002-Fix-doc-install-dir.patch

```diff
@@ -0,0 +1,25 @@
+From 66dec67036116243893fdf74060e55b70f65ab17 Mon Sep 17 00:00:00 2001
```

Comment:
See patch comments above.

Reply:
This patch was created by Bernd Waibel which ensures that the documentation is installed to the correct directory.